src/_imagingft.c

```diff
@@ -819,9 +827,19 @@ font_render(FontObject* self, PyObject* args)
         Py_RETURN_NONE;
     }
 
+    if (strokeWidth) {
+        error = FT_Stroker_New(library, &stroker);
+        if (error)
```

Comment:
Let's defensively use parentheses, even for single-line `if`s

Reply:
All single-line `if`s should now be using parentheses.